docs/proposals/multi-arch-image-mgmt.md

```diff
@@ -0,0 +1,239 @@
+
+# Create a new index by updating an existing index specified by its digest:
+oras manifest index update localhost:5000/hello@sha256:99e4703fbf30916f549cd6bfa9cdbab614b5392fbe64fdee971359a77073cdf9 --add linux-amd64 --remove sha256:fd6ed2f36b5465244d5dc86cb4e7df0ab8a9d24adc57825099f522fe009a22bb
+
+# Merge manifests from the index 'v2-windows' to the index 'v2':
+oras manifest index update localhost:5000/hello:v2 --merge v2-windows
+
+# Update an index and tag the updated index as 'v2.1.0' and 'v2':
+oras manifest index update localhost:5000/hello@sha256:99e4703fbf30916f549cd6bfa9cdbab614b5392fbe64fdee971359a77073cdf9 --add linux-amd64 --tag "v2.1.0" --tag "v2"
+
+# Update an index and save it locally to index.json, auto push will be disabled:
+oras manifest index update --output index.json localhost:5000/hello:v2 --add v2-linux-amd64
+
+# Update an index and output the index to stdout, auto push will be disabled:
+oras manifest index update --output - --pretty localhost:5000/hello:v2 --remove sha256:99e4703fbf30916f549cd6bfa9cdbab614b5392fbe64fdee971359a77073cdf9
+```
+
+### Inspect a multi-arch image
+
+Add two alias `show` and `inspect`:
+
+```bash
+Usage:
+  oras manifest fetch [flags] <name>{:<tag>|@<digest>}
+
+Aliases:
+  fetch, get, show, inspect
+```
+
+## Investigation on other client tools and industry
+
+Most of popular container client tools support create and push a multi-arch image using docker manifest list or OCI image index format, but these tools **require users to push platform-specific image push to the registry separately**. They don’t provide native support for local environment.
+
+- **docker**:
+  - [docker buildx buildx](https://docs.docker.com/reference/cli/docker/buildx/build/)
+  - [docker buildx imagetool](https://docs.docker.com/reference/cli/docker/buildx/imagetools/)
+  - [docker manifest create](https://docs.docker.com/reference/cli/docker/manifest/)
+- **podman (Backed by Red Hat)**: similar with `docker manifest`, it provides `podman manifest` with subcommands to create and manipulate manifest lists and image indexes
+- **crane(Backed by Google)**: provides a single command [crane index append](https://github.com/google/go-containerregistry/blob/main/cmd/crane/recipes.md#create-a-multi-platform-image-from-scratch) to compose an image index
+- **regctl (Individual)** provides [regctl index add/create/delete](https://github.com/regclient/regclient/blob/main/docs/regctl.md#index-commands) to creates or manages OCI Indexes and Manifest Lists
+- **manifest-tool (Individual from AWS):** create docker manifest list or OCI image index in a registry by using the [manifest-tool push command with either a YAML file describing the images to assemble or by using a series of parameters](https://github.com/estesp/manifest-tool?tab=readme-ov-file#sample-usage).
+- **skopeo (Backed by Red Hat)**: doesn’t support OCI image index and docker manifest list
```

Comment:
you may want to consider as well in here Buildah, which I would say positions in between Skopeo and Podman